doc/source/conf.py

```diff
@@ -68,6 +68,17 @@
         '../examples/applications',
         '../examples/developers',
     ]),
+    'binder': {
+        # Required keys
+        'org': 'scikit-image',
+        'repo': 'docs',
+        'branch': 'gh-pages',  # Can be any branch, tag, or commit hash
+        'binderhub_url': 'https://mybinder.org',  # Any URL of a binderhub.
+        'dependencies': '../../requirements/binder/requirements.txt',
+        # Optional keys
+        'filepath_prefix': 'dev/', # A prefix to prepend to filepaths in links.
```

Comment:
I thought we were going for `stable`?

Reply:
(though if you managed to make dev work, all the better!)

Reply:
because for me `dev` is the only part that new commits can modify at the moment: I consider `stable` and other `0.x` directories to be frozen as they have been generated at release time, so I would not like to modify them?